src/index.html

```diff
@@ -14,52 +14,24 @@
     <main class="mainScroll">
       <h1 class="logoTitle">Overlook</h1>
       <section class="userCard">
-        <h2>Hello Salvatore</h2>
-        <h3>Account total: $1000</h3>
-        <button>Book a room</button>
+        <h2 id="user-greeting">Hello Salvatore</h2>
+        <h3 id="account-total">Account total: $1000</h3>
+        <button id="find-a-room">Find a room</button>
       </section>
 
-      <section class="roomList">
-        <h3 class="listHeading">Your Reservations<h3>
-        <section class="roomCard">
-          <h3>Room 1: residential suite</h3>
-          <h4>$666.66 per night</h4>
-          <ul class="roomFeatures">
-            <li>Bed size: king</li>
-            <li>Number of beds: 1</li>
-            <li>Bidet: yup</li>
-          </ul>
-        </section>
-
-        <section class="roomCard">
-          <h3>Room 1: residential suite</h3>
-          <h4>2019/11/30</h4>
-          <ul class="roomFeatures">
-            <li>Bed size: king</li>
-            <li>Number of beds: 1</li>
-            <li>Bidet: yup</li>
-          </ul>
-        </section>
+      <h3 class="listHeading">Your Reservations<h3>
+      <section id="room-list" class="roomList">
 
-        <section class="roomCard">
+        <!-- <section class="roomCard">
           <h3>Room 1: residential suite</h3>
           <h4>2019/11/30</h4>
           <ul class="roomFeatures">
             <li>Bed size: king</li>
             <li>Number of beds: 1</li>
             <li>Bidet: yup</li>
           </ul>
-        </section>
+        </section> -->
 
-        <section class="roomCard">
-          <h3>Room 1: residential suite</h3>
-          <h4>$666.66 per night</h4>
```

Comment:
Are the issues that pop up in WAVE audit for these headers acceptable? Me thinks so but would like second opinion.